js/apps/admin-ui/src/i18n/i18n.ts

```diff
@@ -1,9 +1,11 @@
 import { createInstance } from "i18next";
 import { initReactI18next } from "react-i18next";
 
+import HttpBackend from "i18next-http-backend";
 import environment from "../environment";
 import { joinPath } from "../utils/joinPath";
-import { OverridesBackend } from "./OverridesBackend";
+
+type KeyValue = { key: string; value: string };
```

Comment:
Perhaps we should put this type somewhere in a common section, we often repeat this exact type in other places. But not part of this PR.

Reply:
If it was the same key, value that would make sense but this are translation key value and not attribute key value

Reply:
Yeah, I guess it doesn't hurt having an extra type. Consider this resolved.